test/hermes/error.js

```diff
@@ -37,6 +37,14 @@ e = new Error('another-message');
 print(e);
 //CHECK: Error: another-message
 
+e = new Error(undefined);
+print(e);
```

Comment:
I think we need something like:
```
print(e.hasOwnProperty(e, 'message'));
```

Reply:
There were already some test cases tackling this so I thought it would be unnecessary to repeat them.

Reply:
This is specifically what this PR is doing. It is removing the "message" property if the parameter is `undefined`. We need a test for that specifically.

Reply:
You're right, I don't know how I missed this!
Have you added the test? It appears that the bot closed PR.

Reply:
I believe `e.hasOwnProperty(e, 'message')` always returns `false` whether the message is `undefined` or not.